Lecture-4/Task5/Task5/Application.java

```diff
@@ -0,0 +1,97 @@
+package Task5;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Application {
+
+    private static StringBuffer answer = new StringBuffer("Vlad Bankov");
+    private static StringBuffer guess = new StringBuffer();
+    private static String name = answer.substring(0,answer.indexOf(" "));
+    private static String lastName = answer.substring(answer.indexOf(" ")+1);
+    private static List <String> alreadyGuessedLetters = new ArrayList<>();
+
+    private static void generateGuessPattern(){
+        for (int i = 0; i < answer.length(); i++) {
+            if (String.valueOf(answer.charAt(i)).equals(" ")){
+                guess.append(" ");
+            }else{
```

Comment:
Please, use some formatting styles in code. There's one available in IDEA - `Ctrl + Alt + L` , `Ctrl + Alt + I`